SPECS/python-wheel/python-wheel.spec

```diff
@@ -57,39 +58,49 @@ A Python wheel of wheel to use with virtualenv.
 %prep
 %autosetup -n %{pypi_name}-%{version} -p1
 
-# Empty files make rpmlint sad
-test -s wheel/cli/install.py || echo "# empty" > wheel/cli/install.py
+%generate_buildrequires
+%pyproject_buildrequires
 
 %build
-%py3_build
-
-%if %{without bootstrap}
-%{py3_build_wheel}
-%endif
+%global _pyproject_wheeldir dist
+%python3 -m flit_core.wheel
 
+%pyproject_wheel
 
 %install
-%py3_install
+# pip is not available when bootstrapping, so we need to unpack the wheel and
+# create the entrypoints manually.
+%if %{with bootstrap}
+mkdir -p %{buildroot}%{python3_sitelib}
+unzip %{_pyproject_wheeldir}/%{python_wheel_name} \
+    -d %{buildroot}%{python3_sitelib} -x wheel-%{version}.dist-info/RECORD
+install -Dpm 0755 %{SOURCE1} %{buildroot}%{_bindir}/wheel
+%py3_shebang_fix %{buildroot}%{_bindir}/wheel
+%else
+%pyproject_install
+%endif
+
 mv %{buildroot}%{_bindir}/%{pypi_name}{,-%{python3_version}}
+%if %{with main_python}
 ln -s %{pypi_name}-%{python3_version} %{buildroot}%{_bindir}/%{pypi_name}-3
 ln -s %{pypi_name}-3 %{buildroot}%{_bindir}/%{pypi_name}
-
-%if %{without bootstrap}
-mkdir -p %{buildroot}%{python_wheeldir}
-install -p dist/%{python_wheelname} -t %{buildroot}%{python_wheeldir}
 %endif
 
+mkdir -p %{buildroot}%{python_wheel_dir}
+install -p %{_pyproject_wheeldir}/%{python_wheel_name} -t %{buildroot}%{python_wheel_dir}
+
 
 %check
-rm setup.cfg
-%{python3} -m pip install pluggy more-itertools
-PYTHONPATH=%{buildroot}%{python3_sitelib} py.test3 -v --ignore build
+%if 0%{?with_check}
+# Smoke test
+%{py3_test_envvars} wheel-%{python3_version} version
+%py3_check_import wheel
+%pytest -v --ignore build
+%endif
 
 %files -n python3-%{pypi_name}
 %license LICENSE.txt
 %doc README.rst
-%{_bindir}/%{pypi_name}
-%{_bindir}/%{pypi_name}-3
```

Comment:
Are these two supposed to disappear? Considering the names, they sound like the main component for the package.

Reply:
That's fair, I added it the way they add it in the current version.